### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by vorburger**
As per https://opensource.guide/code-of-conduct/ and generated by https://github.com/apache/fineract/community/code-of-conduct/new.

### CODE REVIEW

**Comment by vidakovic** [comment]
@vorburger can we merge this one? Just asking in case you need a hand...

**Comment by vorburger** [comment]
> @vorburger can we merge this one? Just asking in case you need a hand...

@gstein pointed out by reply on the PMC list that the ASF already has https://www.apache.org/foundation/policies/conduct.html, so I've now removed the text (which was copy/paste from the GitHub template), and instead just link to that now (and mention the PMC email).

PS: In general, you can always :smile_cat:  merge any of my PRs if they LGTY.. (if they are not marked Draft; or if they are big, just _Approve_ but don't merge if you want more reviewers to LGTM them).

**Review by ptuomola — APPROVED** [review]
LGTM

**Pull request merged**
